packages/components/src/components/Link/Link.stories.tsx

```diff
@@ -124,6 +124,11 @@ export const _ExternalLink: Story = {
       onCancel: () => console.log('Analytics event: Canceled'),
       onPress: () => console.log('Analytics event: Pressed'),
       onConfirm: () => console.log('Analytics event: Confirmed'),
+      onOpenURLError: (e, url) => {
+        console.log(JSON.stringify(e))
+        console.log('Error: ' + e)
```

Comment:
Included both because the error object is weird:
- Just straight logging `e` would not log anything to the console (not even that a console log event occurred, literally like no `console.log` was present)
- Concatenating with a string makes it work to log a string-wise error
- The JSON stringify displays the object which seems to vary
   - Sometimes it's a robust object (e.g. `http` or `https` URL) with `message` and other properties
   - Other times it's sparse (e.g. "") with just a `name` (with less detail than the string concatenation above) and `framesToPop` whatever that means

I'm not sure we actually care what this error is, but it seemed to make sense to generally allow consumer efforts to probe it.

Reply:
I like this addition. Should we check `typeof e` to determine whether to log the stringified version or not? i.e. 

```js
console.log(typeof e === 'Object' ? JSON.stringify(e) : `Error: ${e}\`)
```

Reply:
The error output seems... weird.

Like, simultaneous output for a totally empty URL is:
```
 LOG  {"name":"Invariant Violation","framesToPop":1}
 LOG  Error: Invariant Violation: Invalid URL: cannot be empty
 LOG  Error opening URL:
```
where the first is the `JSON.stringify(e)` and second is `"Error: " + e` so the string is seemingly pulling data not present in the object. Maybe it's because it's a JS object and not JSON? But also as noted in the comment above that `console.log(e)` literally breaks anything from logging--it doesn't even have an empty LOG, it just completely prevents the console.log from working.

I didn't dig super deeply into it except [finding this StackOverflow](https://stackoverflow.com/questions/54649465/how-to-do-try-catch-and-finally-statements-in-typescript) on typing it where it also seems pretty arbitrary, although their suggestions are splitting it--but as noted in at least one error it's seemingly both because the string contains data the object doesn't which makes no sense.

As this is in our story and a consumer can send what they like + the observed behavior, didn't seem worth digging deeper into how forcing it to be a string somehow seems to pull out more info than exists in the object form for a case as that clearly makes no sense.

Reply:
Okay, let's keep it as is